openpype/hosts/maya/plugins/publish/collect_yeti_rig.py

```diff
@@ -124,24 +124,26 @@ def get_yeti_resources(self, node):
             image_search_paths = [p for p in
                                   image_search_paths.split(os.path.pathsep) if p]
 
-            # find all ${TOKEN} tokens and replace them with $TOKEN env. variable
-            image_search_paths = self._replace_tokens(image_search_paths)
-
-            # List all related textures
-            texture_filenames = cmds.pgYetiCommand(node, listTextures=True)
-            self.log.debug("Found %i texture(s)" % len(texture_filenames))
+        # find all ${TOKEN} tokens and replace them with $TOKEN env. variable
+        image_search_paths = self._replace_tokens(image_search_paths)
+
+        # List all related textures
+        texture_nodes = cmds.pgYetiGraph(
+            node, listNodes=True, type="texture")
+        texture_filenames = [
```

Comment:
This not being indented now means you can remove the `texture_filenames = []` on line 119.